modules/cms-kit/src/Volo.CmsKit.Common.Application/Volo/CmsKit/Menus/MenuUpdatedHandler.cs

```diff
@@ -0,0 +1,26 @@
+using System.Threading.Tasks;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.Domain.Entities.Events;
+using Volo.Abp.EventBus;
+using Volo.Abp.EventBus.Distributed;
+
+namespace Volo.CmsKit.Menus
+{
+    public class MenuUpdatedHandler : ILocalEventHandler<EntityUpdatedEventData<Menu>>, ITransientDependency
+    {
+        public IDistributedEventBus EventBus { get; }
+
+        public MenuUpdatedHandler(IDistributedEventBus eventBus)
+        {
+            EventBus = eventBus;
+        }
+
+        public async Task HandleEventAsync(EntityUpdatedEventData<Menu> eventData)
+        {
```

Comment:
I didn't understand why we are publishing a distributed event, but not directly using IDistributedCache to remove the cache item, like `await DistributedCache.RemoveAsync(...)`

BTW, if you one day need to publish a distributed event when an entity changes, use [predefined events](https://docs.abp.io/en/abp/latest/Distributed-Event-Bus#pre-defined-events). ABP will automatically publish the event for you.